images/linux/scripts/installers/java-tools.sh

```diff
@@ -57,9 +60,16 @@ rm gradleLatest.zip
 ln -s /usr/share/gradle-"${gradleVersion}"/bin/gradle /usr/bin/gradle
 echo "GRADLE_HOME=/usr/share/gradle" | tee -a /etc/environment
 
+# Install JMeter
+curl -sL https://downloads.apache.org/jmeter/binaries/apache-jmeter-5.2.1.zip -o jmeter.zip
```

Comment:
Please, use parsing of Git release tags to get the latest released version instead of hard-coding the version. 
Something like this should the work (probably not optimized way):
```
jmeterTags="/tmp/jmeter_tags.json"
versions=""

curl -s 'https://api.github.com/repos/apache/jmeter/git/refs/tags' >> $jmeterTags
sorted=$(jq 'sort_by(.ref)' $jmeterTags)

for row in $(echo "$sorted" | jq -c '.[]'); do
    refValue=$(echo "$row" | jq -r 'select( .ref | test("v([0-9]\\.[0-9](\\.[0-9])?$)")) | .ref')
    if [ ! -z "$refValue" ]; then
        versions="${versions}$refValue "
    fi
done

cleanVersion=$(echo "$versions" | rev | cut -d '/' -f 1 | rev)

version=$(echo ${cleanVersion//'v'})
```
the above code outputs the `5.3` which is the latest version now.